### ISSUE TITLE
EVAManager recharges ship when boarding

### ISSUE THREAD

**Issue by mwerle** [toadicus/EVAManager#2]
Steps to reproduce:
- Create a new ship consisting just of a pod and something which drains power (lights, antenna, etc)
- Launch ship (ie, get it on the launchpad)
- Use up some energy.
- Go on EVA -> Power is not removed from ship, EVA Kerbal has 100 units of energy
- Board ship -> Left-over Kerbal Power is added to the ship.

**Comment by mwerle** [comment]
Actually, the issue is more complex.

AntennaRange has a config as follows:

```
EVA_RESOURCE
{
    name = ElectricCharge
    amount = 100
    maxAmount = 100
}
```

With the current implementation, this means every time a Kerbal goes on EVA, it starts with 100 charge (and hence no charge is deducted from the Vessel), however, every time it boards a vessel, it gives its left-over charge back to the vessel.  

So eitherAntennaRange needs to specify FillFromPod as false (not sure if that's possible with the current codebase), or define "amount" as 0.

Down the track, EVAManager needs to be enhanced to only use "amount" as an initialization value, but track the resource properly thereafter.

**Comment by toadicus** [comment]
The "amount = 100" line was an error; changed it to "amount = 0".  Fixed by toadicus/AntennaRange@a888901c.
